tensorflow-frameworks/tensorflow-data/src/test/java/org/tensorflow/data/DataInterfaceTester.java

```diff
@@ -0,0 +1,31 @@
+package org.tensorflow.data;
+
+import org.junit.Test;
+import org.tensorflow.tools.ndarray.IntNdArray;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+import static org.tensorflow.tools.StaticApi.*;
+
+public class DataInterfaceTester {
+  @Test
+  public void testSomething() {
+    IntNdArray matrix2d = ndArrayOfInts(shapeOf(3, 4));
```

Comment:
In [this PR](https://github.com/tensorflow/java/pull/31) again, I dropped the `StaticApi` class, so better not using it :) Meaning that would be `NdArrays.ofInts(Shape.of(3, 4))`

But everything in this class seems to test more `tensorflow-tools` functionalities than `tensorflow-data`, could we simply remove it? Or move it directly to `tensorflow-tools` if you think they are worth it.

Reply:
I agree, I just removed it.